core/controllers/base.py

```diff
@@ -536,7 +584,12 @@ def create_csrf_token(cls, user_id):
 
     @classmethod
     def is_csrf_token_valid(cls, user_id, token):
-        """Validate a given CSRF token with the CSRF secret in memcache."""
+        """Validates a given CSRF token with the CSRF secret in memcache.
```

Comment:
Validates the given CSRF token. (The original docstring was wrong; this has no memcache dependency.)